esp/templates/registration/newuser.html

```diff
@@ -7,7 +7,7 @@
 <div class="note">
   <p>
     <b>Warning: Account not created!</b>  It looks like other accounts already exist for this
-    e-mail address.  Do you already have an {{ site }} account?  If so,
+    email address.  Do you already have an {{ site }} account?  If so,
```

Comment:
Can you mimic what you did for `duplicate_accounts.html`, otherwise the "an" may not agree with the site name.

Reply:
yep!